plugins/search-react/src/components/SearchTracker/SearchTracker.tsx

```diff
@@ -22,7 +22,7 @@ import usePrevious from 'react-use/lib/usePrevious';
 /**
  * Capture search event on term change.
  */
-export const TrackSearch = ({ children }: { children: React.ReactChild }) => {
+export const SearchTracker = ({ children }: { children: React.ReactChild }) => {
```

Comment:
Also, I recommend you deprecate the old name but keep exporting it and add a new export for the new component to avoid breaking changes.

Reply:
I just noticed that this component is not exported in the package, so it's ok to rename without deprecating 🙌🏻